packages/server/modules/auth/services/apps.js

```diff
@@ -30,6 +31,9 @@ module.exports = {
       .select('id', 'name', 'avatar')
       .where({ id: app.authorId })
       .first()
+
+    const defaultApp = getDefaultApp({ id })
```

Comment:
These 2 lines are repeated 3 times, I'd move this out to a reusable function, even if its just at the top of this file and not exported out